fe2-android/app/build.gradle

```diff
@@ -16,179 +16,147 @@ android {
 
         testInstrumentationRunner "com.github.dedis.popstellar.HiltTestRunner"
 
+        // The following feature is not working properly with the coverage on the current build version
+        // As it's purpose is to clear physical data between each tests, its is not used yet.
```

Comment:
```suggestion
        // The purpose of 'clearPackageData: true' is to clear physical data between each tests, but it is not used yet, because it is not working properly with the coverage on the current build version.
```